### ISSUE TITLE
Citation autocomplete does not work when vimtex is located at a path containing spaces

### ISSUE THREAD

**Issue by azag0** [lervag/vimtex#451]
I happen to have `$XDG_DATA_HOME` located at `~/Library/Application Support` and the neovim plugins are there as well. When vimtex is loaded from there, autocomplete of citations does not work (autocomplete of references does work). When vimtex is loaded from a path not containing spaces, the autocomplete does work.

Versions:

- vimtex: 0ff8b2b
- neovim: 0.1.4

`.vimrc`:

```vim
let &rtp  = $XDG_DATA_HOME . '/nvim/plugged/vimtex,' . &rtp
let &rtp .= ',' . $XDG_DATA_HOME . '/nvim/plugged/vimtex/after'

"let &rtp  = '~/.vim/plugged/vimtex,' . &rtp
"let &rtp .= ',~/.vim/plugged/vimtex/after'
```

`preprint.tex`:

```tex
\documentclass{article}
\usepackage{biblatex}
\addbibresource{jh.bib}
\begin{document}
\section{Section}
\label{sec:a}
\cite{Al-SaidiJCTC12}
\cite{}
\ref{}
\printbibliography%
\end{document}
```

`jh.bib`:
```bibtex
@article{AdlerPR62,
 author = {Adler, Stephen L.},
 doi = {10.1103/PhysRev.126.413},
 journal = {Phys. Rev.},
 number = {2},
 pages = {413--420},
 title = {Quantum {{Theory}} of the {{Dielectric Constant}} in {{Real Solids}}},
 url = {http://dx.doi.org/10.1103/PhysRev.126.413},
 volume = {126},
 year = {1962}
}
```

**Comment by lervag** [comment]
Interesting. I will investigate when I get the time, but I suspect this may be a problem also for regular Vim.

**Comment by lervag** [comment]
The problem is, as I suspected, related to the whitespace. In fact, the problem is due to the intermediate style file I use in order to generate the completion candidates. I actually use bibtex in order to generate candidates, and I have a style file located in the vimtex source. Unfortunately, bibtex does _not_ handle whitespace in file names.

I've attempted a fix for this by using an intermediate temporary file. It seems to work here, but I am not sure if the solution is fully portable. Let me know if it works or not.
